src/Illuminate/Database/Schema/Grammars/SQLiteGrammar.php

```diff
@@ -857,4 +857,18 @@ protected function modifyIncrement(Blueprint $blueprint, Fluent $column)
             return ' primary key autoincrement';
         }
     }
+
+    /**
+     * Get the SQL for an unsigned column modifier.
+     *
+     * @param  \Illuminate\Database\Schema\Blueprint  $blueprint
+     * @param  \Illuminate\Support\Fluent  $column
+     * @return string|null
+     */
+    protected function modifyUnsigned(Blueprint $blueprint, Fluent $column)
+    {
+        if ($column->unsigned) {
+            return ' CHECK("'.$column->name.'" >= 0)';
```

Comment:
Please use `wrap()` and lowercase SQL:

```suggestion
            return ' check('.$this->wrap($column->name).' >= 0)';
```